### ISSUE TITLE
dataio - dataio-pyshovel Popup allow all keys (Trac #900)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#894]
Certain keys are being intercepted in the Popup class (particularly j and k).  This is likely an interaction between the global keybindings and Edit.keypress().  A likely fix will involve modifying self._command_map and removing the letters (make sure to copy it before editing, since it is by default a reference to the global keybindings).

Migrated from https://code.icecube.wisc.edu/ticket/900
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T21:45:56", 
    "description": "Certain keys are being intercepted in the Popup class (particularly j and k).  This is likely an interaction between the global keybindings and Edit.keypress().  A likely fix will involve modifying self._command_map and removing the letters (make sure to copy it before editing, since it is by default a reference to the global keybindings).", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547243156161165", 
    "component": "combo core", 
    "summary": "dataio - dataio-pyshovel Popup allow all keys", 
    "priority": "major", 
    "keywords": "dataio-pyshovel", 
    "time": "2015-03-12T05:12:27", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150323T23:30:47: olivas changed milestone from "" to "offline V15-03-00"

**Comment by icecube-trac** [comment]
Trac update at 20150514T03:18:46: 
* olivas changed milestone from "offline V15-03-00" to "offline V15-07-00"
* olivas changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150720T20:46:22: 
* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "accepted" to "closed"
* david.schultz commented:

> In changeset 2170:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2170"
> fix #894
> ```

**Comment by icecube-trac** [comment]
Trac update at 20150810T20:31:17: 
* olivas commented:

> Milestone offline V15-07-00 deleted


* olivas changed milestone from "offline V15-07-00" to ""

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:45:56: nwhitehorn commented:

> In changeset 169145:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169145"
> fix #894
> ```
